packages/manager/modules/pci/src/projects/project/storages/databases/databases.html

```diff
@@ -47,6 +47,14 @@
             ></span>
             <p data-ng-bind=":: $row.id"></p>
         </oui-datagrid-column>
+        <oui-datagrid-column
+            data-property="id"
```

Comment:
does it help customers to show id?

Reply:
Yes, QA told me that it helps them when it comes to billing.